net/src/client/tcp/actors/jsonrpc.rs

```diff
@@ -133,7 +133,18 @@ impl JsonRpcClient {
         );
         let f = socket.execute(&method, params);
 
-        let res = Compat01As03::new(f).await;
+        // TODO: Allow to specify the timeout when starting the actor
+        let timeout_in_millis = 5_000;
+        let res = tokio::time::timeout(
+            Duration::from_millis(timeout_in_millis),
+            Compat01As03::new(f),
+        )
+        .await
+        .map_err(|_err| {
+            log::error!("Failed to connect to {}, will retry later", socket.addr().to_string());
+
+            Error::RequestTimedOut(u128::from(timeout_in_millis))
+        })?;
```

Comment:
I think this change is not needed because the `impl Handler<Request> for JsonRpcClient` already has a timeout parameter.

Reply:
true....it was set to 60 seconds!